pkg/api/controller_test.go

```diff
@@ -2092,6 +2092,37 @@ func TestController_ObjectsStageObjectHandler(t *testing.T) {
 		}
 	})
 
+	t.Run("stage object in storage ns", func(t *testing.T) {
+		linkResp, err := clt.GetPhysicalAddressWithResponse(ctx, "repo1", "main", &api.GetPhysicalAddressParams{Path: "foo/bar2"})
+		verifyResponseOK(t, linkResp, err)
+		if linkResp.JSON200 == nil {
+			t.Fatalf("GetPhysicalAddress non 200 response - status code %d", linkResp.StatusCode())
+		}
+		const expectedSizeBytes = 38
+		resp, err := clt.StageObjectWithResponse(ctx, "repo1", "main", &api.StageObjectParams{Path: "foo/bar2"}, api.StageObjectJSONRequestBody{
```

Comment:
This test will soon break with the limitation of staging object in the namespace. Consider changing the call to Link

Reply:
Update the code to use LinkPhysicalAddress